packages/merge-on-green/src/merge-logic.ts

```diff
@@ -211,8 +211,8 @@ mergeOnGreen.getStatusi = async function getStatusi(
       per_page: 100,
       page: num,
     });
-    if(data[0].context === undefined) {
-      console.info(`no further page data`)
+    if (data[0].context === undefined) {
```

Comment:
Maybe shorten? Same with a few other `=== undefined` and `=== 0` instances below.

```suggestion
    if (!data[0].context) {
```

Reply:
Thank you! I will leave the ```.length === 0``` ones because there are many methods that return an empty array instead of returning falsy.